Libraries/MiscDrivers/ExtMemory/Ext_Flash.h

```diff
@@ -129,6 +129,23 @@ extern "C" {
 #define EXT_FLASH_CMD_DREAD 0xBB /**< Dual SPI Read                  */
 #define EXT_FLASH_CMD_QREAD 0xEB /**< Quad SPI Read                  */
 
+#elif defined(EXT_FLASH_IS25)
+
+#define EXT_FLASH_NAME "IS25"
+
+#define EXT_FLASH_Read_DUMMY \
```

Comment:
Apparently, we carry a legacy typo (lowercase "Read" in the statement "EXT_FLASH_Read_DUMMY"). 
Shall we correct it in this version? @lorne-maxim @sihyung-maxim

Reply:
Nice catch, forgot about this! Let's correct it for all use cases, but let's still include a `#define` to support the legacy name in case there are any projects or library files that still use the legacy name.

```
// Deprecated name. Please use EXT_FLASH_READ_DUMMY
#define EXT_FLASH_Read_DUMMY EXT_FLASH_READ_DUMMY
```

Reply:
While still supporting the legacy name, "EXT_FLASH_Read_DUMMY" replaced with "EXT_FLASH_READ_DUMMY" for all use cases (mx25, is25, w25, mx25_64MB).